canonical-websites/www.ubuntu.com
# 493 build mobile beta nav

## Done

Implement mobile version of beta navigation

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Check the navigation matches the mobile design in the issue


## Issue / Card

Fixes [#493](https://github.com/ubuntudesign/web-squad/issues/493)
